gradle.properties

```diff
@@ -1,8 +1,8 @@
-korkVersion=7.106.0
+korkVersion=7.150.0
+kotlinVersion=1.6.21
```

Comment:
Are we going to run into snags because other repos (e.g. [kork](https://github.com/spinnaker/kork/blob/1a53ec62e94ba42f0e2d23946dc4d71751eec353/gradle.properties#L1), [clouddriver](https://github.com/spinnaker/clouddriver/blob/6c30c181b9f92e0f3c80c5dec547b1c20315246b/gradle.properties#L6)) use kotlin 1.4.x?

Reply:
We're running this version in our production accounts, I haven't seen any snags.

Reply:
I'm still compiling for Kotlin 1.4

Reply:
OK.  We don't use swabbie, and working builds are better than broken builds...